app/src/test/java/com/sdp/movemeet/DistanceCalculatorTest.java

```diff
@@ -52,7 +53,8 @@ public void settingAndGettingIsCorrect() {
                 DUMMY_DATE,
                 DUMMY_DURATION,
                 DUMMY_SPORT,
-                DUMMY_ADDRESS);
+                DUMMY_ADDRESS,
+                DUMMY_CREATION_DATE);
 
         Activity activity2 = new Activity(
```

Comment:
Wouldn't it be possible to create each time those 2 test activities using a for loop to make the test a little bit cleaner?

Reply:
Honestly, I just added this field to all the tests that were already implemented. But i agree that we could refactor that in a cleaner way.

Reply:
Ok ok no problem then, you're right, this test was probably originally created by Mark 😉 We will see if we refactor it this way in the future, but that's not a priority 👍